app/views/layouts/application.html.erb

```diff
@@ -14,6 +14,8 @@
   <body>
     <%= render 'shared/navbar' %>
     <%= render 'shared/flashes' %>
+    <p class="notice"><%= notice %></p>
```

Comment:
I think those are not necessary because there are in the shared/flashers (included in the template of Le Wagon)